source/installation_guide/upgrading/from-2.3-to-3.0.rst

```diff
@@ -38,6 +38,12 @@ New features
 | :ref:`fs_crypt` now has ``maybe`` parameter                | The fs_crypt now requires encryption and decryption keys by default. To keep the old     |
 |                                                            | behavior, add the maybe parameter.                                                       |
 +------------------------------------------------------------+------------------------------------------------------------------------------------------+
+| :dovecot_core:ref:`login_auth_socket_path`                 | Can be used to specify socket to connect in auth process.                                |
++------------------------------------------------------------+------------------------------------------------------------------------------------------+
+| Per-process listeners for protocol processes               | Allows managing connections via UNIX-socket                                              |
++------------------------------------------------------------+------------------------------------------------------------------------------------------+
+| Submission service allows removing received header         | :dovecot_core:ref:`submission_add_received_header`                                       |
++------------------------------------------------------------+------------------------------------------------------------------------------------------+
```

Comment:
Hmm. Why do we have a list of new features in the upgrading page? It doesn't really affect upgrading. And these are rather small features which aren't going to affect all that many people. Shouldn't the new features be in the release notes, and this page only about removed/changed features?

Reply:
My point is that we could somehow get people to use some of these, maybe I went bit overboard here though.